### PR TITLE
article: deploying k3s with command package

### PR DESCRIPTION
**Pull request by rawkode**

### CODE REVIEW

**Review by katcosgrove — APPROVED** [review]
no nits, looks good to me!

**Review by nimbinatus — CHANGES_REQUESTED** [review]
Couple metadata patches to make sure it publishes and a question or two, but looks great!

File: themes/default/content/blog/deploying-k3s-with-command-package/index.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+tags: ["kubernetes"]
+---
+
+We recently announced in our [release blog (66)](https://www.pulumi.com/blog/pulumi-release-notes-66/) a new package: [Command](https://www.pulumi.com/registry/packages/command/). In this article, I want to show you a practical application of this that will allow us to deploy [k3s](https://k3s.io) to a [DigitalOcean](digitalocean.com/products/kubernetes/) droplet. We'll then leverage the Command package to run a remote command to fetch the kubeconfig, generated on the VM, and pull it down to create a Kubernetes provider to deploy nginx.
+
+So, let's get started by deploying our Digital Ocean droplet.
+
+The first thing we're going to do is generate an SSH key for the machine.
+
+```ts
```
[the remark this answers is not in the document]
Comment (rawkode) [reply themes/default/content/blog/deploying-k3s-with-command-package/index.md:17]: Yeah, the how-to will have all the examples. When it's ready, I'll loop back and link from here

**Review by nimbinatus — APPROVED** [review]
Love the image! I'm gonna approve here to unblock; I suspect the docs build will fail because the links in paragraph one aren't relrefs, but let's find out if they have to be :)

**Pull request merged**